cardano-api/cardano-api.cabal

```diff
@@ -123,6 +123,7 @@ library
                       , ouroboros-consensus-shelley
                       , ouroboros-network
                       , ouroboros-network-framework
+                      , plutus-core
```

Comment:
Ideally we should get everything we need via `plutus-ledger-api` and not have to use `plutus-core`. Can you double check if the things are exported via `plutus-ledger-api`, and if not ask for them to be exported via that package.

Reply:
Fixed!